src/pages/AllocationPage/AllocationGive.tsx

```diff
@@ -279,6 +285,72 @@ const AllocationGive = ({
     { success: 'Saved Gifts' }
   );
 
+  const getResult = async (wonderOrgIds: string | any[]) => {
+    let orgIdString = '';
+    let addresses = `&user_addresses=${myUser.address}`;
+    for (let i = 0; i < localGifts?.length; i++) {
+      if (localGifts[i].user.name !== 'Coordinape') {
+        addresses = addresses + `&user_addresses=${localGifts[i].user.address}`;
+      }
+    }
+    for (let i = 0; i < wonderOrgIds?.length; i++) {
+      orgIdString = orgIdString + `&organization_ids=${wonderOrgIds[i]}`;
+    }
+    await fetch(
+      `http://localhost:8001/v1/coordinape/user?${addresses}${orgIdString}&epoch_start=2022-07-21T18:42:57&epoch_end=2022-08-21T18:42:57`,
```

Comment:
can we make this use this only in dev, and a prod or staging url in other envs so we can deploy to our staging env with Vercel?